### ISSUE TITLE
[React Native New Architecture]: Narrator is not announcing any success message on invoking 'Copy to clipboard' button.

### ISSUE THREAD

**Issue by anupriya13** [microsoft/react-native-gallery#586]
ADO 58124627

Build Details:
OS Version: Dev (OS Build 27881.1000)
App: React Native Gallery Experimental
AT: Narrator

 

Repro-Steps:
Launch the React Native Gallery- Experimental app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to 'Home' button and invoke it.
Navigate to 'Clipboard' button and invoke it.
Observe the announcement and behavior.
Actual Result:
On invoking the 'Copy to clipboard' button, Narrator is not announcing any success message.
Note: Visually also no changes appeared on invoking 'Copy to clipboard' button.

Expected Result:
On invoking the 'Copy to clipboard' button, visually the 'Copied' message should appear and Narrator should announce as "Copied to clipboard".

User impact:

Users with visual impairment who rely on screen reader might get confused if narrator does not announce any success message.

MAS Reference:  [MAS 4.1.3 – Status Messages](https://aka.ms/MAS4.1.3)

Trap ID: [1.9 - Feedback Failure](https://aka.ms/FeedbackFailure)
 
“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by anupriya13** [comment]
We need to implement https://github.com/microsoft/react-native-windows/issues/14296 for this to fix, no changes needed in gallery for this

**Comment by anupriya13** [comment]
Doing a workaround and using accessibilityValue for now as the functionality is same.
